library/auth.inc

```diff
@@ -206,27 +206,13 @@ function authCheckExpired($user)
     return false;
 }
 
-function getUserList($cols = '*', $limit = 'all', $start = '0')
-{
-    if ($limit = "all") {
-        $rez = sqlStatement("select $cols from users where username != '' order by date DESC");
-    } else {
-        $rez = sqlStatement("select $cols from users where username != '' order by date DESC limit $limit, $start");
-    }
-
-    for ($iter = 0; $row = sqlFetchArray($rez); $iter++) {
-        $tbl[$iter] = $row;
-    }
-
-    return $tbl;
-}
 
 function getProviderList($cols = '*', $limit = 'all', $start = '0')
```

Comment:
We can remove this function. This one is not used. There is another one with same name in the care coordination module that is used.